openapi.yaml

```diff
@@ -0,0 +1,668 @@
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+        '422':
+          description: Unprocessable Entity
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+    patch:
+      summary: update a service instance
+      tags:
+        - ServiceInstances
+      operationId: serviceInstance.update
+      parameters:
+        - $ref: '#/components/parameters/APIVersion'
+        - $ref: '#/components/parameters/OriginatingIdentity'
+        - name: instance_id
+          in: path
+          description: instance id of instance to update
+          required: true
+          schema:
+            type: string
+        - name: accepts_incomplete
+          in: query
+          description: asynchronous operations supported
+          schema:
+            type: boolean
+      requestBody:
+        description: parameters for the requested service instance update
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ServiceInstanceUpdateRequest'
+      responses:
+        '200': 
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+        '202': 
+          description: Accepted
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+        '400': 
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+        '422': 
+          description: Unprocessable entity
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+    delete:
+      summary: deprovision a service instance
+      tags:
+        - ServiceInstances
+      operationId: serviceInstance.deprovision
+      parameters:
+        - $ref: '#/components/parameters/APIVersion'
+        - $ref: '#/components/parameters/OriginatingIdentity'
+        - name: instance_id
+          in: path
+          description: id of instance being deleted
+          required: true
+          schema:
+            type: string
+        - name: service_id
+          in: query
+          description: id of the service associated with the instance being deleted
+          required: true
+          schema:
+            type: string
+        - name: plan_id
+          in: query
+          description: id of the plan associated with the instance being deleted
+          required: true
+          schema:
+            type: string
+        - name: accepts_incomplete
+          in: query
+          description: asynchronous deprovision supported
+          schema:
+            type: boolean
+      responses:
+        '200':
+          description: OK
```

Comment:
This is missing the operation field that is valid in the response of DELETE on Service Instances

Reply:
yes yes: https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/spec.md#body-9

Reply:
done.